monitoring/dashboards/viya/sas-launched-jobs-node.json

```diff
@@ -1405,14 +1439,25 @@
           "type": "table"
         }
       ],
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
+          },
+          "refId": "A"
+        }
+      ],
       "title": "Active Job Details",
       "type": "row"
     }
   ],
   "refresh": "10s",
-  "schemaVersion": 31,
+  "schemaVersion": 36,
   "style": "dark",
-  "tags": [],
+  "tags": [
+    "sas-viya"
```

Comment:
I'm not a Grafana expert, but this is the only change to this file that I would have expected to have been needed.  Couldn't you just hand-edit the existing JSON file to add this?  I don't know if that would work but...